packages/core/cli/src/commands/init.ts

```diff
@@ -0,0 +1,25 @@
+import log from '../utils/log';
+import getIntegrationsFromPackage from '@vue-storefront/cli/src/utils/getIntegrationsFromPackage';
+import inquirer from 'inquirer';
+import createProject from '../scripts/createProject';
+import path from 'path';
+
+export default async ([projectName]) => {
+  if (!projectName) {
```

Comment:
i think that could be just a first question if not provided (like in vue cli)

Reply:
Just added it with unit test